docs/tutorials/podman_tutorial.md

```diff
@@ -81,6 +81,17 @@ $ ./build.sh
 $ sudo mkdir -p /usr/libexec/cni
 $ sudo cp bin/* /usr/libexec/cni
 ```
+#### Installing CNI config
+Add a most basic network config
+```console
+mkdir -p /etc/cni/net.d
+cat > /etc/cni/net.d/99-loopback.conf <<EOF
```

Comment:
My understanding of CNI from Dan Williams is that loopback device declaration is wholly unnecessary.  I think Matt is on the right track with using the default one.